src/Concordium/Client/Runner.hs

```diff
@@ -4114,6 +4115,23 @@ processIdentityShowCmd action backend =
                     >>= getResponseValueOrDie
                     >>= runPrinter . printAnonymityRevokers . toList
 
+-- | Process a 'consensus ...' command.
+processPoolCmd :: PoolCmd -> Verbose -> Backend -> IO ()
+processPoolCmd action _verbose backend =
```

Comment:
Why did you include the unused argument?

Reply:
Didn't realize to begin with that it wasn't needed and then forgot to remove. Removed now.